packages/kbn-i18n/README.md

```diff
@@ -199,6 +199,20 @@ export const HELLO_WORLD = i18n.translate('hello.wonderful.world', {
 }),
 ```
 
+One more example with a placeholder:
+
+```js
+import { i18n } from '@kbn/i18n';
+
+export function getGreetingMessage(userName) {
+  return i18n.translate('hello.wonderful.world', {
+    defaultMessage: 'Greetings, {name}!',
+    values: { name: <b>userName</b> },
```

Comment:
I'm not sure but it looks like it will not work because we can pass JSX only to intl.formatMessage method or to FormattedMessage component from react-intl. Could you please check?

Reply:
Actually, yes, it won't work unless you pass the result of this function into `innerHTML` or something.